torch/onnx/symbolic_opset11.py

```diff
@@ -435,6 +435,8 @@ def unbind(g, self, dim=0, _outputs=None):
 #          The order is dim_n_begin, dim_n_end, dim_n-1_begin, dim_n-1_end, ..., dim_m_begin, dim_m_end,
 #          where m is in range [0, n].
 def _prepare_onnx_paddings(g, dim, pad):
+    if not sym_help._is_packed_list(pad) and sym_help._is_list(pad) and sym_help._is_scalar_list(pad):
```

Comment:
Does it well handle empty list case?

Reply:
Yes, this works for empty tensors as well. 
Though ORT currently cannot handle empty input list.